config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # get '/users' => 'users#index'
+  # get '/users/:id', to: 'users#show'
```

Comment:
Goodjob so far it is important to keep your codebase as clean as possible. Kindly remove unused code for it can cause errors. Commented code makes your code base look untidy.

Reply:
Thanks for your review @KayLemba, I forgot to remove those lines, I'll remove them immediately.